modules/setting/user.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package setting
+
+import (
+	"strings"
+
+	"code.gitea.io/gitea/modules/container"
+)
+
+const (
+	UserDeletionKey = "deletion"
+)
+
+// userSetting represents user settings
+type userSetting struct {
+	content container.Set[string]
```

Comment:
It's quite misleading. When you read `User.content.Contains` you might think it is for "enabling something". But, it is disabling something.